web/style.css

```diff
@@ -192,61 +249,29 @@ footer span {
 
 a {
 	text-decoration: none;
+	color: var(--accent);
 }
 
 a:link {
 	border-bottom: 1px dotted;
-	color: #387e8e;
 }
 
 a:visited {
 	border-bottom: 1px dotted;
-	color: #387e8f;
 }
 
 a:link:hover,
 a:visited:hover {
 	border-bottom: 1px solid;
 }
 
-@media (prefers-color-scheme: dark) {
-	body,
-	body.disconnected,
-	body.dialling,
-	body.error {
-		background: #801384;
-		background: radial-gradient(ellipse at bottom, #801384 0%, #170926 100%);
-		background-image: url(stars.svg);
-		background-position: center center;
-		background-repeat: no-repeat;
-		background-size: cover;
-		color: #80d5d7;
-	}
-
-	body.connected {
-		background: radial-gradient(#170926 0%, #3a0c3c 100%);
-		background-image: url(wormhole.svg);
-		background-position: center center;
-		background-repeat: no-repeat;
-		background-size: cover;
-		border-radius: 20px;
-		border: 4px dashed #ccc;
-	}
-
-	body.highlight {
-		background: radial-gradient(#801384 0%, #170926 100%);
-		border: 4px dashed #fcabff;
-	}
-
-	a:link {
-		color: #b58787;
-	}
-
-	a:visited {
-		color: #b58788;
+/* Styling for screen widths 700px and upwards. */
+@media only screen and (min-width: 700px) {
```

Comment:
we select background colour (the list is hardcoded in main.js, i've been meaning to move it to css) based on the derived key, as an extra mitm check.

the colours show here beautifully on wide screens, but not on narrow ones. can we think of a way of keeping that feature on smaller windows and mobile? maybe give the container div a translucent background? or somehow expose a band of that colour?

Reply:
This crossed my mind too, yeah. There isn't any change at all when you successfully connect etc. I can look at some different ways to show that colour more when the background is less visible

Reply:
I've pushed a change to make the container have some space at the top and bottom for the coloured background to show through. Let me know what you think of it :)

Reply:
![image](https://user-images.githubusercontent.com/21127383/103756214-f5468f00-5006-11eb-86c0-ae6906206e04.png)